projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -1047,3 +1026,95 @@ export function runExternalSpartacusLibrary(
 function createModuleFileName(config: FeatureConfig): string {
   return `${dasherize(config.moduleName)}-feature.module.ts`;
 }
+
+/**
+ * Used to sort the features in the correct order.
+ */
+function calculateSort(libraryA: string, libraryB: string): number {
+  const indexA = installationOrder.indexOf(libraryA);
+  const indexB = installationOrder.indexOf(libraryB);
+
+  /**
+   * In case a feature module is _not_ found in the `installationOrder`,
+   * we want to sort it at the end of the list.
+   */
+  return (indexA > -1 ? indexA : Infinity) - (indexB > -1 ? indexB : Infinity);
+}
+
+export function orderInstalledFeatures<T extends LibraryOptions>(
+  options: T
+): Rule {
+  return (tree: Tree, context: SchematicContext): void => {
+    context.logger.info('Ordering the installed Spartacus features...');
+
+    const basePath = process.cwd();
+    const { buildPaths } = getProjectTsConfigPaths(tree, options.project);
+    for (const tsconfigPath of buildPaths) {
+      const { appSourceFiles } = createProgram(tree, basePath, tsconfigPath);
+
+      let spartacusFeaturesModule: SourceFile | undefined;
+      for (const sourceFile of appSourceFiles) {
+        if (
+          sourceFile
+            .getFilePath()
+            .includes(`${SPARTACUS_FEATURES_MODULE}.module.ts`)
+        ) {
+          spartacusFeaturesModule = sourceFile;
+          break;
+        }
+      }
+
+      if (!spartacusFeaturesModule) {
```

Comment:
why do we continue if there's no spartacus features module found

I'm assuming it's to keep the schematics running right, and not break right?

Reply:
We continue because we are iterating over the `tsconfigPaths`, and the module might be in some other project.